File: android/src/main/java/com/authsignal/react/RequestMetadata.kt
package com.authsignal.react

import com.authsignal.AuthsignalRequestMetadata

internal object RequestMetadata {
  fun configure() {
    AuthsignalRequestMetadata.setWrapperSDK(
      sdk = "react-native",
      version = BuildConfig.VERSION_NAME,
      userAgentToken = "AuthsignalReactNativeSDK",
    )
  }
}

File: ios/RequestMetadata.swift
import Authsignal
import Foundation

private let authsignalReactNativeVersion = "3.3.1"

enum RequestMetadata {
  static func configure() {
    AuthsignalRequestMetadata.setWrapperSDK(
      "react-native",
      version: authsignalReactNativeVersion,
      userAgentToken: "AuthsignalReactNativeSDK"
    )
  }
}
